src/components/settings/SettingsAppearanceTab/ThemeCard.jsx

```diff
@@ -1,40 +1,65 @@
 import PropTypes from 'prop-types';
-import { Card, Form } from '@edx/paragon';
+import { Button, Card, Form } from '@edx/paragon';
 import ThemeSvg from './ThemeSvg';
+import { SCHOLAR_THEME } from '../data/constants';
+
+export default function ThemeCard({
+  themeVars, selected, setTheme, openCustom,
+}) {
+  const isCustom = themeVars.title === 'Custom Theme';
```

Comment:
nit: Is it worth putting this string in a constant `CUSTOM_THEME_LABEL` or something to avoid having to repeat it twice (and in case a test needs it in the future, it could be worth exporting it as a named export as well, i.e. `export const CUSTOM_THEME_LABEL`).